pxr/imaging/hdSt/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(PXR_PACKAGE hdSt)
 
 # XXX We tmp check for PXR_ENABLE_GL_SUPPORT since Storm currently still only
 # runs on OpenGL. Once Storm uses only Hgi, remove GL_SUPPORT check.
-if (NOT ${PXR_BUILD_GPU_SUPPORT} OR NOT ${PXR_ENABLE_GL_SUPPORT})
+if (NOT ${PXR_BUILD_GPU_SUPPORT} OR NOT (${PXR_ENABLE_GL_SUPPORT} OR APPLEIOS))
```

Comment:
instead of not APPLEIOS should this be a test on METAL_SUPPORT instead?

Reply:
Agree, modified accordingly